homeassistant/components/plugwise/config_flow.py

```diff
@@ -179,8 +183,15 @@ async def async_step_user(
                 errors[CONF_BASE] = "invalid_setup"
             except InvalidAuthentication:
                 errors[CONF_BASE] = "invalid_auth"
-            except PlugwiseException:
+            except (
+                ClientError,
+                ConnectionFailedError,
+                InvalidXMLError,
+                ResponseError,
+            ):
```

Comment:
I do not fully understand this change. Could you elaborate on why this is using more specific errors instead of their parent `PlugwiseException` now?

Reply:
That is not correct IMHO. There are multiple errors not caught this way. For example, the `connect` call can raise an `UnsupportedDeviceError`, which is now no longer caught.

Reply:
Tnx, I'll double-check and rework this tonight.

Reply:
All exceptions should now be lined up between the library and the integration.